x-pack/plugins/spaces/public/nav_control/nav_control_popover.tsx

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import './nav_control.scss';
```

Comment:
This doesn't seem like the appropriate file to import this from. This file is for a popover, but the `nav_control.scss` file contains styles for the whole page?